multiversx_sdk/abi/managed_decimal_value.py

```diff
@@ -0,0 +1,103 @@
+import io
```

Comment:
Same comments as for `ManagedDecimalSignedValue`.

Reply:
applied the same changes here, as well.